src/main/java/sortBench/algorithms/select/SelectSort.java

```diff
@@ -0,0 +1,25 @@
+package sortBench.algorithm
+
+public class SelectSort {
```

Comment:
replace 
`public class SelectSort {`
by
`public class SelectSort implements ISortingAlgorithm {`

Reply:
Well the push steps works for now....